cmd/edgectl/aes_install.go

```diff
@@ -818,11 +741,17 @@ func (i *Installer) Quit() {
 	i.cancel()
 }
 
-func (i *Installer) ShowWrapped(text string) {
-	text = strings.Trim(text, "\n")                  // Drop leading and trailing newlines
-	for _, para := range strings.Split(text, "\n") { // Preserve newlines in the text
-		for _, line := range doWordWrap(para, "", 79) { // But wrap text too
-			i.show.Println(line)
+func (i *Installer) ShowWrapped(texts ...string) {
```

Comment:
```suggestion
// ShowWrapped displays to the user (via the show logger) the text items passed
// in with word wrapping applied. Leading and trailing newlines are dropped in
// each text item (to make it easier to use multiline constants), but newlines
// within each item are preserved. Use an empty string item to include a blank
// line in the output between other items.
func (i *Installer) ShowWrapped(texts ...string) {
```

Reply:
My comment here renders weirdly in GitHub, so let me clarify. All I'm trying to do is suggest a doc comment for `ShowWrapped`. I wrote this comment in my branch, but I think it makes more sense in yours, since you added the variadic arguments (brilliant!).

Reply:
Ok, I will add this comment.